### PR TITLE
Task: run e2e flow for scaffolding-cli in containers

### PR DESCRIPTION
**Pull request by estherthetesteramido**
#### 📲 What

boostrap the node projects in the pipeline, just a like a real person.

#### 🤔 Why
		
So we have confidence things continue to work even if no one uses it!

#### 🛠 How
		
Containers, config file, being awesome.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import {MenuItem, TextField, makeStyles} from "@material-ui/core"
-import {getLanguages} from "../../lib/contentful-api"
```
Comment (naren-bellala) [inline packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx:2]: Why is this removed?

Comment (estherthetesteramido) [reply packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx:2]: Ah, in my branch it was unused. Good shout, adding back in.

File: packages/scaffolding-cli/src/domain/config/worker_maps/shared.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ ${ranAdvanced ? advanced : basic}
 
 🤓 To get started: open ${dir}/README.md
 
-📖 For guides and supporting information see: <url>
+📖 For guides and supporting information see: https://amido.github.io/stacks/
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/worker_maps/shared.ts:98]: 👍 also there is a lib that makes links clickable in the terminal of certain types we can wrap it in that

File: tests/packages/scaffolding-cli/node-projects/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- To build the project `docker run -it --rm scaffolding-cli npm run build`
+
+All the project package scripts can be executed as sanity checks.
+
+### Interactive
+
+Runs interactively, with entry point being the project root directory (`PROJECT_WORKDIR`).
+
+`docker build --build-arg PROJECT_WORKDIR=<PROJECT_WORKDIR> --build-arg CONFIG_FILENAME=<CONFIG_FILENAME> . -t scaffolding-cli`
+`docker run -it --rm scaffolding-cli /bin/bash`
```
Comment (dnitsch) [inline tests/packages/scaffolding-cli/node-projects/README.md:40]: don;t you want to mount a volume back to the host so people can see the output?

File: packages/scaffolding-cli/src/domain/config/worker_maps/shared.ts (hunk 2 above)
Comment (estherthetesteramido) [reply packages/scaffolding-cli/src/domain/config/worker_maps/shared.ts:98]: Good shout, mine does this automatically? Think most terminals do if prefixed with http/https?

File: tests/packages/scaffolding-cli/node-projects/README.md (hunk 3 above)
Comment (estherthetesteramido) [reply tests/packages/scaffolding-cli/node-projects/README.md:40]: Good shout!
